packages/rum-core/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -1040,3 +1040,72 @@ describe('rum track custom timings', () => {
     expect(warnSpy).toHaveBeenCalled()
   })
 })
+
+describe('track hasReplay', () => {
+  let setupBuilder: TestSetupBuilder
+  let handler: jasmine.Spy
+  let getViewEvent: (index: number) => View
+
+  beforeEach(() => {
+    ;({ handler, getViewEvent } = spyOnViews())
```

Comment:
weird `;` Should be at the end not ?

Reply:
No, prettier inserts a semicolon when the line starts with a paren to avoid ASI issues

See: https://github.com/prettier/prettier/issues/1935#issuecomment-305968119